### DISCUSSION TITLE
Option to show shared album activity on photos in timeline

### DISCUSSION THREAD

**Discussion by fivestones** [immich-app/immich#4812] [Feature Request]
### The feature

Currently to see shared album activity, one has to be viewing the photo as part of a shared album. I'd like to have the option to show shared activity on the photo when viewing in the main timeline.

If the photo has shared album activity for multiple albums, in the side view that shows the shared album activity, there could be a heading with the name of the first shared album, then the activity, then another shared album heading, and the activity, and so on.

@martabal @jrasm91 

Thanks for considering!

### Platform

- [ ] Server
- [x] Web
- [x] Mobile

**Comment by spencerdf1992** [comment]
I'm new to the software and figured was trying to figure out where the setting is to enable this feature... Guess your request answers my question.

I'm trying to use Immich to share my "live" family album with immediate family. New photos within shared albums with mobile notifications to those that want them would be an amazing addition to Immich.

**Comment by joshuajeschek** [comment]
I think this could be an option for shared albums, specific to users. So user A could turn on "show in timeline" for album 1, and only see those photos as part of their timeline, while user B might have it disabled, or enable it for one or more other albums.

**Comment by flightmansam** [comment]
I think this would be an excellent feature.

**Comment by nathanfredericks** [comment]
I'd love to see this feature added

**Comment by GJurriens** [comment]
I would also like this feature.
And additionally, cause this could be related... is is also possible to have the people recognition work in shared albums as well? Cause currently when looking up a person, I only get results out of my own photo's, not also the photo's in the shared albums as well.

**Comment by nidbCN** [comment]
I also have this demand. I have many album of travels. Some of them I want share to my family, some not. But my shared not display in timeline, If I share the whole account it can be display on timeline but that's not what I want.

**Comment by 4f1sh3r** [comment]
I think the same. The complete partner sharing is too much for us. On album level it's perfect, but they should be integrated as if they were local photos. So including map and face recognition etc... Ideally as an option per album.

**Comment by dhunter49** [comment]
I find it really useful, because when we were using The-App-That-Shall-Not-Be-Named we had 15 GB of storage **EACH** and now we have all the storage for all of our family members/users.
So when we had 15gb each we could just send and download our pics, now it's kind of useless to copy those photos across different user accounts.

**Comment by ALMAAAAA** [comment]
Landed here, because I have the same issue as multiple other people. Having the option to toggle a button on shared album, that makes the pictures appear in the timeline would be the perfect solution for me.

**Comment by rexkani** [comment]
I have the same request. Also using immich for family album sharing, family members are only viewers and this feature would make their app experience more organic

**Comment by denis-tzdm** [comment]
I am a new user of immich, and the product is so perfect that I was even surprised to find that such a feature is missing. It is definitely a must for shared family albums.
Sharing the entire collection does not allow you to choose, and the option to display the shared album in the timeline would not break the overall user experience.

**Comment by RobinS15** [comment]
It's still not added right?
I waited long, but it's still not added yet, I guess.

**Comment by j-land3** [comment]
Crazy that this is still missing. The only reason I don’t use immich. 🤞🙏🏻🛐
